Corgibytes.Freshli.Cli/Program.cs

```diff
@@ -24,25 +31,66 @@ public static async Task<int> Main(string[] args)
                 .InvokeAsync(args);
         }
 
+        private static void ConfigureLogging(string consoleLogLevel, string logfile)
+        {
+            NLog.LogLevel desiredLevel = NLog.LogLevel.FromString(consoleLogLevel);
+
+            LoggingConfiguration config = new LoggingConfiguration();
+            if (IsNullOrEmpty(logfile))
+            {
+                ColoredConsoleTarget consoleTarget = new ColoredConsoleTarget();
+                config.AddTarget("console", consoleTarget);
+                consoleTarget.Layout = DefaultLogLayout;
+                config.LoggingRules.Add(new LoggingRule("*", desiredLevel, consoleTarget));
+            }
+            else
+            {
+                FileTarget fileTarget = new FileTarget();
+                config.AddTarget("file", fileTarget);
+                fileTarget.FileName = logfile;
+                fileTarget.Layout = DefaultLogLayout;
+                config.LoggingRules.Add(new LoggingRule("*", desiredLevel, fileTarget));
+            }
+
+            LogManager.Configuration = config;
+        }
+
```

Comment:
What do you think about moving the logger setup to it's own class?

Reply:
I created #219 to track this request.